job_templates/service_action_-_ansible_default.erb

```diff
@@ -12,7 +12,7 @@ template_inputs:
     At least one of state and enabled are required. Note that reloaded will start
     the service if it is not already started, even if your chosen init system wouldn't
     normally.
-  options: "started\r\nstopped\r\nrestarted\r\nreloaded"
+  options: "started\r\nstopped\r\nrestarted\r\nreloaded
```

Comment:
Missing double quotes at the end of the line

Reply:
good catch, thank you